### PR TITLE
AISDK-206: Add Language Parameter to streaming client

### PR DESCRIPTION
**Pull request by kshiflett88**

### CODE REVIEW

File: src/rev_ai/models/streaming/mediaconfig.py [hunk 1]
```diff
@@ -28,4 +30,5 @@ def get_content_type_string(self):
             (';layout={}'.format(self.layout) if self.layout else '') + \
             (';rate={}'.format(self.rate) if self.rate else '') + \
             (';format={}'.format(self.format) if self.format else '') + \
-            (';channels={}'.format(self.channels) if self.channels else '')
+            (';channels={}'.format(self.channels) if self.channels else '') + \
+            (';language={}'.format(self.language) if self.language else '')
```
Comment (amikofalvy) [inline src/rev_ai/models/streaming/mediaconfig.py]: This does not work. The language parameter is not part of the content_type parameter.

`language` needs to be it's own query parameter

**Review by amikofalvy — CHANGES_REQUESTED** [review]
The approach does not work

**Review by amikofalvy — CHANGES_REQUESTED** [review]
Approach looks good. We need to bump the version number for release.

File: src/rev_ai/models/streaming/mediaconfig.py (hunk 1 above)
Comment (kshiflett88) [reply src/rev_ai/models/streaming/mediaconfig.py]: > Approach looks good. We need to bump the version number for release.

@amikofalvy for clerification do I change the `current_version` in `setup.cfg` from 2.16.0 to 2.17.0 and then add a note in the `HISTORY.rst` file about the change to the version?

**Review by amikofalvy — APPROVED** [review]

**Comment by amikofalvy** [comment]
After you merge this you will need to add a changelog entry on the docs site as well. Like this one for the NodeSDK https://github.com/revdotcom/revai-docs/pull/141

**Pull request merged**